.github/workflows/CI-nightly.yaml

```diff
@@ -111,9 +111,7 @@ jobs:
           cd build && git checkout ${GH_REF}
           cat requirements-dev.txt | grep tox | xargs python3 -m pip -qq install
 
-          if [[ 'ppc64le' == $(uname -m) ]]; then
-            sed -i "s/PyYAML==5.4.1/PyYAML==6.0.1/" requirements.txt
-          else
+          if [[ 's390x' == $(uname -m) ]]; then
```

Comment:
@modassarrana89 @yussufsh this PR updates PyYAML to 6.0.1 in requirements.txt; does this change look correct?